app/assets/stylesheets/components/common/_global_nav_content.scss

```diff
@@ -0,0 +1,223 @@
+// Global Navigation
+//
+// Styleguide Global Navigation
+
+$max-cols: 6;
+$gutter: $default-gutter / 2;
+
+.global-nav__content {
```

Comment:
If this is the name of the new file (the new 'block') should this be `.global-nav-content`?